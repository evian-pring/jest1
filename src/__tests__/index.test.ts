import { actions } from '../index';

test('actions', () => {
  const result = actions.meth1();
  expect(result).toMatchSnapshot();

});
