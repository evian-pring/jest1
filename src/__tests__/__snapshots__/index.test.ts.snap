// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actions 1`] = `"hello"`;
